release/202008/metrics-head-rc.md

```diff
@@ -0,0 +1,51 @@
+# CHAOSS Metrics  
+### Upcoming Release 202008 [Under Review]
+
+CHAOSS metrics are identified and defined using a continuous release process. The metrics are officially released biannually following a 30 day comment period. To contribute to the release or comment on metrics under review, please follow the links provided in the working group tables located below.
+
+To obtain a pdf copy of previous releases or see what is new in this release please visit the [release notes page](https://chaoss.community/release-history/).
+
+To make suggestions or edits to this webpage please visit the [website repo](https://github.com/chaoss/website) and open an issue or create a pull request.
+
+#### Disclaimer
+
+Released metrics are only a subset of metric ideas that are being developed. CHAOSS makes no assertions about the completeness of this list. If you would like to learn more about metrics, suggest new metrics, and or help define metrics please visit the working group repositories.  
```

Comment:
@ElizabethN pointed out that the text was misleading. Here is an alternative proposal.
```suggestion
Metrics on this page were debated in working groups and underwent a review period. CHAOSS acknowledges that more metrics exist and would like to add them in the future. If you would like to learn more about metrics, suggest new metrics, and or help define metrics please visit the working group repositories.  
```

Reply:
Yeah, I know the language on this needs to change a bit. The sentiment is accurate though. Will try a rewrite.